plugin.manifest

```diff
@@ -1,7 +1,7 @@
 pluginname=OpenCast
 pluginclassname=OpenCast
 origin=elan-ev
-version=2.5.3
+version=2.5.5
```

Comment:
what happend to v2.5.4 ? 🧐 
why should the version be bumped before the release? 😅

Reply:
2.5.4 has been used for testing...

Reply:
I expected the next release to be 2.5.4

Reply:
We expect a lot of things to work as we think they should, but life is full of wonder! 😅

Reply:
BTW, to clear things out here! I made these changes as a PR and not merged it directly! Therefore, the final decision is to be made by Till! So no worries, it could be changed!
I bumped the version so to notify that this PR carries lots of critical changes that needs the next big version.

Reply:
So we have to drop a number for symoblic reasons?
I don't think a version change should be included in a merge.